### PR TITLE
Add version history for 5.1.9 (rebased onto develop)

### PR DESCRIPTION
**Pull request by sbesson**
This is the same as gh-2336 but rebased onto develop.

----

Preliminary draft for the version history. Please feel free to point out edits/corrections.

@melissalinkert @jburel @sbesson @joshmoore

### CODE REVIEW

**Comment by sbesson** [comment]
--rebased-from #2336

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+       - fixed null pointer exceptions while parsing metadata
+   - Leica LIF
+       - fixed large multi-tiled files from having incorrect plane offsets after the 2GB mark
+   - EM formats (MRC and Spider)
+       - added native length support for EM readers
+   - Gatan
+       - fixed erroneous metadata parsing
+       - added support for parsing and storing the physical sizes in native units
+   - OME-TIFF
+       - improved handling of OME-TIFF multi-file fileset’s with partial metadata blocks
```
Comment (mtbc) [inline docs/sphinx/about/whats-new.txt]: "filesets"?

**Comment by mtbc** [comment]
Minor grocer quibble but good to merge either way.

**Comment by sbesson** [comment]
Thanks. Merging and we can correct the spelling in an upcoming PR.

**Comment by hflynn** [comment]
Bah, that one slipped passed me :(

**Pull request merged**